content/commands/ts.add/index.md

```diff
@@ -177,6 +178,30 @@ This override is effective only for this single command and does not set the tim
 This argument has no effect when a new time series is created by this command.
 </details>
 
+<details open><summary><code>IGNORE ignoreMaxTimediff ignoreMaxValDiff</code></summary> 
+
+is the policy for handling insertion of samples that may be duplicates of existing samples according to the following rules:
+
+  - `ignoreMaxTimeDiff`: the system considers a new insertion a duplicate if the time difference does not exceed the maximum allowed. This value defaults to the `IGNORE_MAX_TIME_DIFF` configuration parameter.
```

Comment:
Please explain these two parameters together. It is considered a duplicate only if both conditions hold: "A new sample is considered duplicate only if ... and ... 

Also, consider not using "the system".